Makefile

```diff
@@ -18,7 +18,7 @@ setup: ## Clean and Install npm dependencies
 ci: lint test
 
 lint:  ## Run linting of code.
-	npm run lint:nofix
+	npm run lint
```

Comment:
Won't this possibly allow GH actions to change code according to lint results? Why change it?

Reply:
Currently, lint won't fix anything before we add the plugin for it. I've already created a ticket for that when I was working in Create UI.